views/Home/components/HomeCertificates/index.tsx

```diff
@@ -29,48 +39,94 @@ const App: NextPage<Props, any> = ({ t }) => {
             fontSize="lg"
             mt="4"
             fontWeight="medium"
-            color={useColorModeValue('gray.600', 'whiteAlpha.700')}
+            color={useColorModeValue("gray.600", "whiteAlpha.700")}
           >
             {t("home-text33")}
           </Text>
         </Box>
-        {language === 'zh-CN' && (
-          <Stack spacing="0" direction={{ base: 'column', md: 'row' }} justifyContent="space-around">
-            <Box textAlign="center" as="a" href="/zh/cert/huawei-stack-8.0" target="_blank">
-              <Image src="https://api7-website-1301662268.file.myqcloud.com/logo/logo-huawei-compatible2.png" maxH="130px" />
+        {language === "zh-CN" && (
+          <Stack
+            spacing="0"
+            direction={{ base: "column", md: "row" }}
+            justifyContent="space-around"
+          >
+            <Box
+              textAlign="center"
+              as="a"
+              href="/zh/cert/huawei-stack-8.0"
+              target="_blank"
+            >
+              <Image src="/static/images/huawei-compatible.jpeg" maxH="80px" />
```

Comment:
center the image would be better
![image](https://user-images.githubusercontent.com/2561857/122010364-a2f76e80-cded-11eb-9383-6ece654f4ac2.png)

```suggestion
              <Image src="/static/images/huawei-compatible.jpeg" maxH="80px"  margin="0 auto"/>
```